WP_CLI_CS/ruleset.xml

```diff
@@ -77,28 +77,6 @@
 
 		<!-- Keep short ternaries around for WP-CLI. -->
 		<exclude name="Universal.Operators.DisallowShortTernary.Found"/>
-
-		<!-- Allow filesystem operations because WordPress APIs may not be available -->
-		<exclude name="WordPress.WP.AlternativeFunctions.file_system_operations_chgrp" />
-		<exclude name="WordPress.WP.AlternativeFunctions.file_system_operations_chmod" />
-		<exclude name="WordPress.WP.AlternativeFunctions.file_system_operations_chown" />
-		<exclude name="WordPress.WP.AlternativeFunctions.file_system_operations_fclose" />
-		<exclude name="WordPress.WP.AlternativeFunctions.file_system_operations_file_put_contents" />
-		<exclude name="WordPress.WP.AlternativeFunctions.file_system_operations_fopen" />
-		<exclude name="WordPress.WP.AlternativeFunctions.file_system_operations_fputs" />
-		<exclude name="WordPress.WP.AlternativeFunctions.file_system_operations_fread" />
-		<exclude name="WordPress.WP.AlternativeFunctions.file_system_operations_fsockopen" />
-		<exclude name="WordPress.WP.AlternativeFunctions.file_system_operations_fwrite" />
-		<exclude name="WordPress.WP.AlternativeFunctions.file_system_operations_is_writable" />
-		<exclude name="WordPress.WP.AlternativeFunctions.file_system_operations_is_writeable" />
-		<exclude name="WordPress.WP.AlternativeFunctions.file_system_operations_mkdir" />
-		<exclude name="WordPress.WP.AlternativeFunctions.file_system_operations_pfsockopen" />
-		<exclude name="WordPress.WP.AlternativeFunctions.file_system_operations_readfile" />
-		<exclude name="WordPress.WP.AlternativeFunctions.file_system_operations_rmdir" />
-		<exclude name="WordPress.WP.AlternativeFunctions.file_system_operations_touch" />
-		<exclude name="WordPress.WP.AlternativeFunctions.rename_rename" />
-		<exclude name="WordPress.WP.AlternativeFunctions.unlink_unlink" />
```

Comment:
If you remove these last two as well, then you need to add the `rename` and `unlink` groups to the list of `exclude`s.

Reply:
The error code is build up of `group_name_function_name`, so you can always tell the group based on the error code ;-)